ecs-cli/modules/cli/compose/project/project_parseV3_test.go

```diff
@@ -110,6 +116,12 @@ func TestParseV3WithOneFile(t *testing.T) {
 			IpAddress: aws.String("10.0.0.0"),
 		},
 	}
+	mysqlCon.HealthCheck = &ecs.HealthCheck{
+		Command:  aws.StringSlice([]string{"CMD-SHELL", "curl -f http://theecscliisawesome.com || exit 1"}),
```

Comment:
```
$ whois theecscliisawesome.com
No match for "THEECSCLIISAWESOME.COM".
>>> Last update of whois database: 2018-06-28T05:36:09Z <<<
```

I'd recommend using either a real domain that we control (like "https://ecs.aws") or using one of the IANA-managed example domains ("https://example.com").